arviz/plots/densityplot.py

```diff
@@ -86,36 +86,36 @@ def plot_density(
     hdi_markers : str
         A valid `matplotlib.markers` like 'v', used to indicate the limits of the highest density
         interval. Defaults to empty string (no marker).
-    shade : Optional[float]
+    shade : float, optional
         Alpha blending value for the shaded area under the curve, between 0 (no shade) and 1
         (opaque). Defaults to 0.
-    bw: Optional[float or str]
+    bw : float or str, optional
         If numeric, indicates the bandwidth and must be positive.
         If str, indicates the method to estimate the bandwidth and must be
         one of "scott", "silverman", "isj" or "experimental" when `circular` is False
         and "taylor" (for now) when `circular` is True.
         Defaults to "default" which means "experimental" when variable is not circular
         and "taylor" when it is.
-    circular: Optional[bool]
+    circular : bool, optional
         If True, it interprets the values passed are from a circular variable measured in radians
         and a circular KDE is used. Only valid for 1D KDE. Defaults to False.
     grid : tuple
         Number of rows and columns. Defaults to None, the rows and columns are
         automatically inferred.
-    figsize : Optional[Tuple[int, int]]
+    figsize : int, optional
```

Comment:
```suggestion
    figsize : (float, float), optional
```

Using `int` was incorrect in the original docs. The `Tuple[int, int]` is translated to `(int, int)`, however, due to `int` being wrong, the correct type for figsize should be the suggestion above: `(float, float)`